Pipfile

```diff
@@ -13,16 +13,14 @@ numpy = "*"
 psychrolib = "*"
 pythermalcomfort = "*"
 icecream = "*"
-pydantic = ">=2.8.2
-plotly = ">=5.23.0"
-scipy = ">=1.14.0
+pydantic = "*"
 
 [dev-packages]
 pytest-playwright = "*"
 pytest-xdist = "*"
 bump-my-version = "*"
 black = "*"
 
-
```

Comment:
do not commit empyt lines.

Reply:
This is because I duplicated the code of some files by deleting them as a result